tutorial/tutorial.py

```diff
@@ -40,7 +40,7 @@
 from cubitpy.cubit_utility import get_surface_center
 
 
-def cubit_step_by_step_tutorial_cli(input_file_path, *, display=True):
+def cubit_step_by_step_tutorial_cli(input_file_path, *, display=True, cubit=None):
```

Comment:
Is it really necessary to pass the `cubit` object from the outside here? From a testing point of view it should be fine to have this function create it, since in the end we don't test a state of `cubit` but test the created input file.

Reply:
To use the compare function already implemented, one has to pass the cubit initialization. Trying to do so without passing it to the tutorial causes the comparison to fail as it does not read the cubit elements created by the tutorial.